assets/src/edit-story/migration/migrations/test/v0018_defaultBackgroundElement.js

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import createDefaultBackgroundElement from '../v0018_defaultBackgroundElement';
+
+describe('createDefaultBackgroundElement', () => {
+  it('should remove background id', () => {
+    const migrated = createDefaultBackgroundElement({
+      pages: [
+        {
+          backgroundElementId: 'x',
+          elements: [{ id: 'x' }],
+        },
+      ],
+    });
+    const page = migrated.pages[0];
+    expect(page).not.toHaveProperty('backgroundElementId');
+  });
+
+  it('should mark existing background shapes as default background', () => {
```

Comment:
Nit: should mark only the first shape as default background

Reply:
Only one element _can_ be background, though?

Reply:
Well, I know. It's just for the test naming purposes, to ensure that the migration would only try to mark the first existing shape as the default background. Not that important though.